gen_messages.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+
+import yaml
+import glob
+import json
+import os
+import sys
+
+if len(sys.argv) < 2:
+    print("Usage: gen_messages.py path-to-messages")
+    sys.exit(1)
+
+datafiles = sys.argv[1]
+
+
+def u_constrt(loader, node):
+    return node.value
+def io_constrt(loader, node):
+    value = loader.construct_mapping(node)
+    return value
+def list_constrt(loader, node):
+    value = loader.construct_mapping(node)
+    return value
+def obj_constrt(loader, node):
+    value = loader.construct_mapping(node)
+    return value
+def str_constrt(loader, node):
+    return node.value
+def vec3_constrt(loader, node):
+    return node.value
+def color_constrt(loader, node):
+    return node.value
+yaml.add_constructor('!u', u_constrt)
+yaml.add_constructor('!io', io_constrt)
+yaml.add_constructor('!color', color_constrt)
+yaml.add_constructor('!list', list_constrt)
+yaml.add_constructor('!obj', obj_constrt)
+yaml.add_constructor('!str64', str_constrt)
+yaml.add_constructor('!str32', str_constrt)
+yaml.add_constructor('!str256', str_constrt)
+yaml.add_constructor('!vec3', vec3_constrt)
+
+dirs = ['EventFlowMsg', 'DemoMsg']
+
+for dir in ['EventFlowMsg', 'DemoMsg']:
+    if not os.path.exists(f"msg/{dir}"):
```

Comment:
You can use Path (import from pathlib) to simplify this.

```py
(Path("msg") / dir).mkdir(parents=True, exist_ok=True)
```

Reply:
Ok